index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Home | CSCS</title>
+    <link rel="stylesheet" href="https://rawcdn.githack.com/anonymoustaco/collegestationcodesociety.github.io/b57ed23d7334cfb26ae5095e43e279a0c4563211/main.css">
```

Comment:
With the links, leave them as local to the website (relative paths), rather than linking to the github files. It makes the site more portable.
```suggestion
    <link rel="stylesheet" href="main.css">
```